### PR TITLE
Add restroom

### PR DESCRIPTION
**Pull request by eunbigo91**
Please review this, @casey-collab

### CODE REVIEW

File: korean/restroom/restroom.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+화장실에서 할 일을 다 마쳤어요. 가볍고 시원한 기분으로 무엇을 할까요?
+
+[국회로 간다](filibuster/filibuster.md)?
```
Comment (casey-collab) [inline korean/restroom/restroom.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

File: korean/restroom/restroom.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+[노래를 부른다](sing-a-song/song.md)?
```
Comment (casey-collab) [inline korean/restroom/restroom.md]: This link is also invalid.

File: korean/restroom/restroom.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+[벽으로 돌진한다](rush/rush.md)?
```
Comment (casey-collab) [inline korean/restroom/restroom.md]: This link is also invalid.

File: korean/restroom/restroom.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+
+[어쩔줄 모르고 그냥 운다](cry/cry.md)?
```
Comment (casey-collab) [inline korean/restroom/restroom.md]: This link is also invalid.

File: korean/restroom/restroom.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+
+[방을 둘러본다](look-around/look-around.md)?
```
Comment (casey-collab) [inline korean/restroom/restroom.md]: This link is also invalid.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by eunbigo91** [comment]
Please review this, @casey-collab

**Comment by casey-collab** [comment]
Looks great! :ice_cream:

**Pull request merged**